src/cf-typography/src/licensed-fonts.less

```diff
@@ -0,0 +1,103 @@
+/*
+ * Source: http://fast.fonts.net/cssapi/44e8c964-4684-44c6-a6e3-3f3da8787b50.css
+ * This file has been edited to use absolute URLS so we can concatenate it with
+ * all of our other styles.
+ */
+@import url(//fast.fonts.net/t/1.css?apiType=css&projectid=44e8c964-4684-44c6-a6e3-3f3da8787b50);
```

Comment:
Let's add a comment for this line to indicate its purpose and that it should not be removed. How about:

```less
// Pings Fonts.com for pageview tracking. Must be used even when self-hosting the fonts.
```

Reply:
Presume this should be fonts.net.